src/com/stuypulse/stuylib/control/feedforward/ArmFeedforward.java

```diff
@@ -0,0 +1,61 @@
+/* Copyright (c) 2023 StuyPulse Robotics. All rights reserved. */
+/* This work is licensed under the terms of the MIT license */
+/* found in the root directory of this project. */
+
+package com.stuypulse.stuylib.control.feedforward;
+
+import com.stuypulse.stuylib.control.Controller;
+import com.stuypulse.stuylib.streams.filters.IFilter;
+
+/**
+ * A feedforward term to account for gravity for motorized arms.
+ *
+ * <p>The motor feedforward used in the context of an arm will not account for gravity that is
+ * acting on the arm.
+ *
+ * <p>Can be paired with MotorFeedforward or other controllers with .add
+ *
+ * @author Myles Pasetsky (myles.pasetsky@gmail.com)
+ */
+public class ArmFeedforward extends Controller {
+
+    /** voltage to hold arm horizontal */
+    private final Number kG;
+
+    /**
+     * function to configure units of cosine (or even use sin if angles are measured differently)
+     */
+    private final IFilter cosine;
+
+    /**
+     * Create arm feedforward
+     *
+     * @param kG term to hold arm vertical against gravity (volts)
+     */
+    public ArmFeedforward(Number kG) {
+        this(kG, Math::cos);
```

Comment:
this is in radians by default, idk if you want that

Reply:
made it degrees